src/main/java/seedu/address/model/person/DateOfBirth.java

```diff
@@ -50,6 +51,29 @@ public DateOfBirth(String date) {
         this.isEmpty = false;
     }
 
+    /**
+     * Returns true if DateOfBirth is empty, false otherwise.
+     * @return boolean
+     */
+    public boolean isEmpty() {
+        return this.isEmpty;
+    }
+
+    /**
+     * Takes in the {@code DateOfBirth} argument as well as a boolean to signify if "NA" can be taken as an argument
+     * to return an empty {@code DateOfBirth}, returns a newly constructed {@code DateOfBirth} with the arguments given.
+     * @param dob
+     * @param isNaAllowed
+     * @return DateOfBirth
```

Comment:
Might want to fill in descriptions of the `@params`? Otherwise should remove them so that header comment doesn't look incomplete.

Perhaps change `{@code DateOfBirth}` to `{@code String dob}` to distinguish it from a `DateOfBirth` object?

```suggestion
     * Takes in the {@code String dob} argument as well as a boolean to signify if "NA" can be taken as an argument
     * to return an empty {@code DateOfBirth}, returns a newly constructed {@code DateOfBirth} with the arguments given.
     * @param dob
     * @param isNaAllowed
     * @return DateOfBirth
```